site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -216,26 +242,598 @@ They can do the same for Chrome on desktop by going to
+   "approved_clients": ["abc", "def", "ghi"],
+  }]
+}
+```
+
+If the user is not signed in, respond with HTTP 401 (Unauthorized).
+
+The returned accounts list is consumed by the browser and will not be
+available to the RP.
+
+#### Client metadata endpoint {: #client-metadata-endpoint }
+
+The IdP's client metadata endpoint returns the relying party's metadata such as
+the RP's privacy policy and terms of service. RPs should provide links to their
+privacy policy and terms of service to the IdP in advance. These links are
+displayed in the sign-in dialog when the user hasn't registered on the RP with
+the IdP yet.
+
+The browser sends a `GET` request using the `client_id`
+[`navigator.credentials.get`](#sign-into-rp) without cookies. For example:
+
+```http
+GET /metadata.php?client_id=1234 HTTP/1.1
+Host: idp.example
+Referer: https://rp.example/
+Accept: application/json
+Sec-Fetch-Dest: webidentity
+```
+
+The properties for the client metadata endpoint include:
+
+<table class="with-heading-tint with-borders">
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>privacy_policy_url</code> (optional)</td>
+    <td>RP privacy policy URL.</td>
+  </tr>
+  <tr>
+     <td><code>terms_of_service_url</code> (optional)</td>
+     <td>RP terms of service URL.</td>
+  </tr>
+</table>
+
+The browser expects a JSON response from the endpoint:
+
+```json
+{
+  "privacy_policy_url": "https://rp.example/privacy_policy.html",
+  "terms_of_service_url": "https://rp.example/terms_of_service.html",
+}
+```
+
+The returned client metadata is consumed by the browser and will not be
+available to the RP.
+
+#### ID assertion endpoint {: #id-assertion-endpoint }
+
+The IdP's ID assertion endpoint returns an assertion for their signed-in user.
+When the user signs in to an RP website using [`navigator.credentials.get()`
+call](#sign-into-rp), the browser sends a `POST` request with cookies and a
+content type of `application/x-www-form-urlencoded` to this endpoint with the
+following information:
+
+<table class="with-heading-tint with-borders">
+  <thead>
+    <tr>
+      <th>Property</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tr>
+    <td><code>client_id</code> (required)</td>
+    <td>The RP's client identifier.</td>
+  </tr>
+  <tr>
+     <td><code>account_id</code> (required)</td>
+     <td>The unique ID of the signing in user.</td>
+  </tr>
+  <tr>
+     <td><code>nonce</code> (optional)</td>
+     <td>The request nonce, provided by the RP.</td>
+  </tr>
+  <tr>
+     <td><code>disclosure_text_shown</code></td>
+     <td>Results in a string of <code>"true"</code> or <code>"false"</code> (rather than a boolean). The result is <code>"false"</code> if the disclosure text was not shown. This happens when the RP's client ID was included in the <code>approved_clients</code> property list of the response from the <a href="#accounts-list-endpoint">accounts list endpoint</a>.</td>
```

Comment:
It could also happen if the browser has observed a sign-up moment in the past in the absence of `approved_clients`.